Makefile

```diff
@@ -321,6 +321,10 @@ generate_node_state_machine_diagram: ## (Re)generates the Node State Machine dia
 test_all: ## Run all go unit tests
 	go test -p 1 -count=1 ./...
 
+.PHONY: test_e2e
+test_e2e: ## Assumes that `localnet_up` is running *warm*.
```

Comment:
1. Try running `make` from the root of the repo. The comment is actually the `help` command and not just a makefile comment.

2. **Optional NIT:** Is there a way to add a small check to validate that it's up and running? Potentially another target making a small RPC call? See `docker_check` as an example